adder/util.go

```diff
@@ -142,6 +142,21 @@ func Pin(ctx context.Context, rpc *rpc.Client, pin *api.Pin) error {
 	)
 }
 
+// AddNodeLocal puts the given node to the local peer.
+func AddNodeLocal(ctx context.Context, rpc *rpc.Client, node ipld.Node) error {
```

Comment:
as noted above, this is duplicate code not really needed when you have a working blockAdder that does this.